prepare yaml file to use macro expand

opcli opened the pull request:
The yaml file will expand by simple pattern replacement.

Adding the config.yaml to servo.yaml.

ekalosak reviewed (approved):
lgtm
note: we could use some CI

Commit:
Prepar the yaml file for macro expansion.

The macro expansion will just use search and replace base
on a dictionary. e.g. using bash we can create the config file as:

cat conf <<EOF
ACCOUNT=abc.com
EOF

Expand the macro we can do:
. conf
sed -i -e "s/@@ACCOUNT@@/$ACCOUNT/g" k8s/servo.yaml

Commit:
Adding config.yaml to servo.yaml

Without config.yaml. On starting the servo.yaml will complian
can't find the config.yaml
